fabric_plugin/tasks.py

```diff
@@ -368,9 +399,12 @@ def run_script(ctx,
             env[CTX_SOCKET_URL] = proxy.socket_url
             env['LOCAL_{0}'.format(CTX_SOCKET_URL)] = proxy.socket_url
             files.upload_env_script(env)
-            command = 'cd {0} && source {1} && {2}'.format(
-                cwd, files.remote_env_script_path, command)
-            run = conn.sudo if use_sudo else conn.run
+            if python_prefix:
+                command = 'cd {0} && source {1} && {2} {3}'.format(
+                    cwd, files.remote_env_script_path, python_prefix, command)
```

Comment:
there is already a `command_prefix` in this function
we should use that instead, ie. if it's not set, only then figure out the python executable automatically

Reply:
Done